docs/docsource/reference/apis/iam/create_access_key_using_vaultclient.rst

```diff
@@ -0,0 +1,35 @@
+Create a First AccessKey Using vaultclient
+==========================================
+
+The first set of credentials of an account is created using
+``vaultclient`` and admin credentials. Later on, account credentials
+can be used with AWS API to create users and policies.
+
+Prerequisites
+-------------
+
+- The data service is deployed
+- Admin credentials 
+- Admin routes
+
+Procedure
+---------
+
+#. Set the admin access key ID and secret access key.
+
+   .. code::
+
```

Comment:
Before this I forgot this command:
```
    VAULT_CONTAINER=kubectl --kubeconfig=/etc/kubernetes/admin.conf  -n zenko get pods  -l app.kubernetes.io/name=connector-vault | sed -n '2 p' | awk '{print $1}'
```

Reply:
I am not going to say for sure but anytime I use kubectl + sed/awk there is a better option with `-o jsonpath`. Just a thought.

Reply:
Is this command listing the pods?

Reply:
You are right, how about this?
```
    VAULT_CONTAINER=$(kubectl --kubeconfig=/etc/kubernetes/admin.conf -n zenko get pods -l app.kubernetes.io/name=connector-vault -o jsonpath='{.items[0].metadata.name}''
```